R/grade_code.R

```diff
@@ -83,13 +83,24 @@ grade_code <- function(
   allow_partial_matching = getOption("gradethis.code.partial_matching", TRUE),
   glue_correct = getOption("gradethis_glue_correct"),
   glue_incorrect = getOption("gradethis_glue_incorrect"),
-  glue_pipe = getOption("gradethis_glue_pipe"),
+  glue_pipe = deprecated(),
   grader_args = deprecated(),
   learnr_args = deprecated()
 ) {
   ellipsis::check_dots_empty()
   if (is_present(grader_args)) deprecate_warn("0.2.0", "grade_code(grader_args = )")
   if (is_present(learnr_args)) deprecate_warn("0.2.0", "grade_code(learnr_args = )")
+  if (is_present(glue_pipe)) {
+    deprecate_warn(
+      "0.2.1",
```

Comment:
@schloerke Is this value okay? If so I'd bump gradethis to `0.2.1.9000` before merging.

Reply:
Sounds good